### ISSUE TITLE
NTR: Fungus CSF test positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8056]
New term request Fungus CSF test positive (samant_100930004411): Fungi; Candida; visually (India ink) or culture positive; cerebrospinal fluid  UMLS: 1142108

**Comment by pnrobinson** [comment]
@MickeySegal
I think we would want to specify the method, e.g., PCR, culture, serology ?

**Comment by pnrobinson** [comment]
or possible a generic term such as Fungal bloodstream infection HP:0034749

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000729
- label: Positive CSF fungus test
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: 
- def: Any kind of test for a fungal infectious agent in the cerebrospinal fluid (CSF) positive.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8056
- synonyms:
